uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/src/main/java/org/uberfire/ext/security/management/keycloak/BaseKeyCloakManager.java

```diff
@@ -102,12 +102,20 @@ protected void loadConfig( final ConfigProperties config ) {
     }
 
     protected synchronized Keycloak getKeyCloakInstance() {
-        if (this.keycloak == null) {
+        if ( null == this.keycloak ) {
```

Comment:
Yes, looks like a left-over, @romartin? Also can you add some JavaDoc why this method is synchronized?